spec/lib/adaptor.spec.js

```diff
@@ -57,7 +57,16 @@ describe("Adaptor", function() {
       );
     });
 
-    context("if no pin is specified", function() {
+    context("if opts are provided", function() {
+      it("throws an error", function() {
+        var fn = function() { return new Adaptor(); };
+        expect(fn).to.throw(
+          "No port specified for Sphero adaptor 'undefined'. Cannot proceed"
+        );
+      });
+    });
```

Comment:
Nope, one tests if only port is not provided, the other tests when no `opts` at all are provided. Behaviour varies slightly in the code.